packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -483,6 +501,41 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
 
     prepareForCommit(root.containerInfo);
 
+    // Invoke instances of getSnapshotBeforeUpdate before mutation.
+    nextEffect = firstEffect;
+    // TODO Start new commit phase timer from ReactDebugFiberPerf
+    while (nextEffect !== null) {
```

Comment:
Why are we adding a new traversal of the effect list instead of using the host effect traversal? (`commitAllHostEffects`)?

Reply:
Yeah, good call. Initial approach was just whatever I could get working quickly. Planning on iterating.